src/cloud/components/Views/index.tsx

```diff
@@ -76,6 +75,20 @@ export const ViewsManager = ({
     )
   )
 
+  useEffect(() => {
```

Comment:
I think we can just use memo hook. Could you explain why did you use effect hook?